src/drivers/intel/alh.c

```diff
@@ -44,6 +44,8 @@ static int alh_set_config(struct dai *dai, struct sof_ipc_dai_config *config)
 
 	alh->params.stream_id = config->alh.stream_id;
```

Comment:
I wonder if we also need to do some changes to i.MX DAIs (drivers/imx/sai.c, drivers/imx/esai.c)

Reply:
@dbaluta It depends whether you support SMP. It can be changed along with adding multicore support to IMX.

Reply:
@tlauda no i.MX integration has SMP support for now. So, this change only affects SMP configurations right?

Reply:
@dbaluta Yes.